owtf/webapp/src/containers/Report/Table.js

```diff
@@ -212,6 +224,25 @@ export class DataTable extends React.Component {
           <Pane border padding={10}>
             <Heading>MORE DETAILS</Heading>
             {/* {output} */}
+            { /**Display only when resourcelist is more than 0,else output.Resourcelist = undefined will appear in UI  */
+              (resourceList.length > 0) &&
+              <Table
+                border
+                padding={10}
+                margin={5}
+              >
+                {'ResourceList'}
+                <ul>
+                  {resourceList.map(
+                    (resource, index) => <li key={index}>
+                      <a href={resource[1]} target="__blank">
+                        {resource[0]}
+                      </a>
+                    </li>
+                  )}
+                </ul>
```

Comment:
Use Evergreen-ui's **UnorderedList** component here.

Reply:
Thank You, Will fix it!!